collectors/diskspace.plugin/README.md

```diff
@@ -2,6 +2,33 @@
 
 This plugin monitors the disk space usage of mounted disks, under Linux.
 
+Two charts are available for every mount:
+ - Disk Space Usage
+ - Disk Files (inodes) Usage
+
+## configuration
+
+Simple patterns can be used to exclude mounts from showed statistics based on path or filesystem. By default read-only mounts are not displayed. To display them `yes` should be set for a chart instead of `auto`.
```

Comment:
This doesn't where I need to set the yes. You probably want to mention it after the first quoted fragment.

Reply:
You can set it for `space usage for all disks` and/or `inodes usage for all disks` as well.